### DISCUSSION TITLE
Is it possible to use this plugin with portable Git?

### DISCUSSION THREAD

**Discussion by anareaty** [Vinzent03/obsidian-git#504] [Q&A]
I can't install Git on my work computer, so I use portable Git on usbdrive instead. But this plugin don't recognize it. Maybe there are some configurations I can change to make it work? I am very new to all the Git related stuff, and also English is not my native language, so I could be missing something and would really appreciate any help.

**Comment by Vinzent03** [comment]
There is a setting called custom git binary path, which you can set to your portable git installation path.
